module1/lessons/boolean_logic.md

```diff
@@ -140,6 +153,32 @@ Let's revisit that last expressions in `pry`, but let's add some parentheses.
 false && false || true
 false && (false || true)
 ```
+
+<section class="dropdown">
+
+### Expressions in JavaScript
+
+These expressions will also work in JavaScript:
+
+```javascript
```

Comment:
I like the addition of showing this in practice with a function in JS!  It's kind of like a 'peek and build' example :)